redirects/linux.nginx

```diff
@@ -0,0 +1,344 @@
+}
+
+location /minio/baremetal/replication/replication-overview.html {
+     return 301 https://min.io/minio/docs/linux/administration/bucket-replication.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/bucket-replication-overview.html {
+     return 301 https://min.io/minio/docs/linux/administration/bucket-replication.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/replication/enable-server-side-one-way-bucket-replication.html {
+     return 301 https://min.io/minio/docs/linux/administration/bucket-replication/enable-server-side-one-way-bucket-replication.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/replication/enable-server-side-two-way-bucket-replication.html {
+     return 301 https://min.io/minio/docs/linux/administration/bucket-replication/enable-server-side-two-way-bucket-replication.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/replication/enable-server-side-multi-site-bucket-replication.html {
+     return 301 https://min.io/minio/docs/linux/administration/bucket-replication/enable-server-side-one-way-bucket-replication.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/replication/server-side-replication-resynchronize-remote.html {
+     return 301 https://min.io/minio/docs/linux/administration/bucket-replication/server-side-replication-resynchronize-remote.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/replication/site-replication-overview.html {
+     return 301 https://min.io/minio/docs/linux/operations/install-deploy-manage/multi-site-replication.html?ref=docs-redirect;
+}
+
+# Monitoring Redirects from NuDocs 1 to NuDocs 2
+# ==============================================
+
+location /minio/baremetal/monitoring/monitoring-overview.html {
+     return 301 https://min.io/minio/docs/linux/operations/monitoring.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/monitoring/metrics-alerts/minio-metrics-and-alerts.html {
+     return 301 https://min.io/minio/docs/linux/operations/monitoring.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/monitoring/metrics-alerts/collect-minio-metrics-using-prometheus.html {
+     return 301 https://min.io/minio/docs/linux/operations/monitoring/collect-minio-metrics-using-prometheus.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/monitoring/logging/minio-logging.html { 
+     return 301 https://min.io/minio/docs/linux/monitoring/minio-logging.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/monitoring/healthcheck-probe.html {
+     return 301 https://min.io/minio/docs/linux/operations/monitoring/healthcheck-probe.html?ref=docs-redirect;
+}
+
+# Bucket Notification Redirects from NuDocs 1 to NuDocs 2
+# =======================================================
+
+location /minio/baremetal/monitoring/bucket-notifications/bucket-notifications.html {
+     return 301 https://min.io/minio/docs/linux/administration/monitoring.html?ref=docs-redirect;
```

Comment:
https://min.io/docs/minio/linux/administration/monitoring/bucket-notifications.html might be better.